include/adm/elements/audio_block_format_matrix.hpp

```diff
@@ -83,6 +83,8 @@ namespace adm {
     ADM_EXPORT void set(Rtime rtime);
     /// @brief Duration setter
     ADM_EXPORT void set(Duration duration);
+    /// @brief InitializeBlock setter
+    ADM_EXPORT void set(InitializeBlock initializeBlock);
```

Comment:
could add this to auto base instead?

Reply:
8026acc1668bff7eaa674d0e10b5b63749e21d9c